assets/quick-order-list.js

```diff
@@ -123,7 +122,7 @@ class QuickOrderList extends HTMLElement {
       .then((response) => response.text())
       .then((responseText) => {
         const html = new DOMParser().parseFromString(responseText, 'text/html');
-        const sourceQty = html.querySelector(this.quickOrderListId);
+        const sourceQty = html.querySelector(`#${this.quickOrderListId}`);
```

Comment:
Hey Sofia, good question. We need this changes because otherwise it's not working. When we use `querySelector` we pass an `ID` and an `ID` should be wrapped with `' '`. It's hard to see when this affects the form. If change the cart to drawer and try to change qty in the drawer you will see that the form won't be updated because `html.querySelector(this.quickOrderListId);` will be `null`.

Reply:
Interesting 🤔 I havent looked into this yet, but doesnt that mean the existing quick order list would be broken?

Reply:
It doesn't because we changed it everywhere accordingly. 

I leave a short video and I'm sorry for this complication. https://screenshot.click/09-14-ckarz-cba3c.mp4